lib/common/output_xml.c

```diff
@@ -559,3 +560,18 @@ pcmk__output_xml_peek_parent(pcmk__output_t *out) {
     /* If queue is empty NULL will be returned */
     return g_queue_peek_tail(priv->parent_q);
 }
+
+void
+pcmk__set_xml_subformat(pcmk__output_t *out, enum pcmk__xml_subformat fmt)
+{
+    private_data_t *priv = NULL;
+
+    CRM_ASSERT(out != NULL && out->priv != NULL);
```

Comment:
for consistency with best practices, parenthesize the boolean expressions here and in the remainder of the commits:
`CRM_ASSERT((out != NULL) && (out->priv != NULL));`

https://clusterlabs.org/pacemaker/doc/2.1/Pacemaker_Development/singlehtml/index.html#operators

Reply:
Changed, even though I find this style needlessly verbose, especially for simple things like this.